docs/stepactions.md

```diff
@@ -21,8 +21,8 @@ weight: 201
   - [Controlling Step Execution with when Expressions](#controlling-step-execution-with-when-expressions)
 
 ## Overview
-> :seedling: **`StepActions` is an [beta](additional-configs.md#beta-features) feature.**
-> The `enable-step-actions` feature flag must be set to `"true"` to specify a `StepAction` in a `Step`.
+> **`StepActions` is a stable feature.**
+> Step actions are enabled by default. You can disable them by setting the `enable-step-actions` feature flag to `"false"`.
```

Comment:
NIT: it may be ok to provide a grace period where folks can still disable `StepActions`, but eventually the flag should be completely removed. We don't support disabling API-driven GA features; users can not use them in the API. If an admin wants to prevent certain APIs from being used, they need to do that via some admission controller logic or similar.

Reply:
I would rather have the flag removed then if that is the case, considering that StepActions are going into GA.

Reply:
yeah that's valid, I will ensure that this is a no op and remove the rest of the implementation of the feature flag then, in a later release it will have to be removed